cogito/put.go

```diff
@@ -16,7 +16,7 @@ type Putter interface {
 	// ProcessInputDir validates and extract the needed information from the "put input".
 	ProcessInputDir() error
 	// Sinks return the list of configured sinks.
```

Comment:
and a error. Either change the comment to includes returns an error, or write the comment differently to say what the method does.

Reply:
I have updated the comment, `Sinks()` validate also if a specific sink is supported.